flamedisx/x1t_sr1.py

```diff
@@ -0,0 +1,107 @@
+"""XENON1T SR1 implementation
+"""
+import numpy as np
+import tensorflow as tf
+
+from multihist import Hist1d
+import wimprates
+
+import flamedisx as fd
+
+export, __all__ = fd.exporter()
+
+o = tf.newaxis
+
+##
+# Yield maps
+##
+
+
+s1_map, s2_map = [
+    fd.InterpolatingMap(fd.get_resource(fd.pax_file(x)))
+    for x in ('XENON1T_s1_xyz_ly_kr83m-SR1_pax-664_fdc-adcorrtpf.json',
+              'XENON1T_s2_xy_ly_SR1_v2.2.json')]
+
+
+##
+# Flamedisx sources
+##
+
+
+@export
```

Comment:
Do we need @export here? I guess only on SR1ERSource and SR1NRSource or do we use SR1Source directly?

Reply:
We might for development in notebooks for example for the NRSource. Don't we,  @JelleAalbers ?

Reply:
For developing we can just do `from flamedisx.x1t_sr1 import SR1Source` right? Or does the export mechanism prevent that somehow? I guess we don't want users to see `SR1Source` since its not fully specified.

Reply:
Both approaches seem valid. I tend to lean towards exporting most things myself, but we can do either. Currently we're not very consistent in flamedisx (eg SR0Source is exported, as is SR0WIMPSource, but SR0ERSource is not..), so maybe it's easiest to just go through this once after both of your PRs are merged.